src/Resources/config/services.xml

```diff
@@ -13,6 +13,9 @@
             <argument type="service" id="service_container" />
             <argument /> <!-- argument added dynamically -->
         </service>
+	
+	<!-- Add service alias for autowiring -->
+	<service id="KnpU\OAuth2ClientBundle\Client\ClientRegistry" alias="knpu.oauth2.registry" public="false" />
```

Comment:
Per the official 3.3 UPGRADE documentation, they recommend they do it the way I have it in the PR.

```
<service id="annotations.reader" class="Doctrine\Common\Annotations\AnnotationReader" public="false" />
<service id="Doctrine\Common\Annotations\Reader" alias="annotations.reader" public="false" />
```